packages/expo-image-picker/ios/EXImagePicker/EXImagePicker.m

```diff
@@ -220,13 +220,18 @@ - (void)handleImageWithInfo:(NSDictionary * _Nonnull)info
 {
   NSURL *imageURL = [info valueForKey:UIImagePickerControllerReferenceURL];
   NSDictionary *metadata = [info objectForKey:UIImagePickerControllerMediaMetadata];
-  UIImage *image;
+  UIImage *image = [info objectForKey:UIImagePickerControllerOriginalImage];
+  image = [self fixOrientation:image];
   if ([[self.options objectForKey:@"allowsEditing"] boolValue]) {
-    image = [info objectForKey:UIImagePickerControllerEditedImage];
-  } else {
-    image = [info objectForKey:UIImagePickerControllerOriginalImage];
+    NSValue* rectVal = [info objectForKey:UIImagePickerControllerCropRect];
+    CGRect rect = rectVal.CGRectValue;
```

Comment:
Is it possible? 🤔 
```suggestion
    CGRect rect = [info objectForKey:UIImagePickerControllerCropRect].CGRectValue;
```
And if it's not:
```suggestion
	CGRect rect = ((NSValue *) [info objectForKey:UIImagePickerControllerCropRect]).CGRectValue;
```

Reply:
First one isn't possible ;P You need to cast explicitly.